package.json

```diff
@@ -36,11 +39,14 @@
   },
   "homepage": "https://github.com/mapbox/assembly#readme",
   "dependencies": {
+    "@mapbox/batfish": "^0.5.0",
     "autoprefixer": "^6.6.1",
     "concat-with-sourcemaps": "^1.0.4",
     "cp-file": "^4.1.1",
     "csso": "^3.1.1",
+    "es6-promise": "^4.1.1",
```

Comment:
This should be automatically loaded by Batfish: https://github.com/mapbox/batfish/blob/a50eea4fbd1b21f4f11bc7e63111e43769d6b9ad/lib/create-webpack-config-client.js#L22